pkg/util/pki/csr.go

```diff
@@ -166,6 +168,27 @@ func BuildKeyUsages(usages []v1.KeyUsage, isCA bool) (ku x509.KeyUsage, eku []x5
 	return
 }
 
+func BuildKeyUsagesKube(usages []certificatesv1.KeyUsage) (ku x509.KeyUsage, eku []x509.ExtKeyUsage, err error) {
+	var unk []certificatesv1.KeyUsage
+	if len(usages) == 0 {
+		usages = []certificatesv1.KeyUsage{certificatesv1.UsageDigitalSignature, certificatesv1.UsageKeyEncipherment}
```

Comment:
I _think_ `usages = append(usages, v1.DefaultKeyUsages()...)` (what is used in `BuildKeyUsages`) is different to this list here - namely, `serverAuth` is included. I may be wrong and we changed this? :)

Reply:
No, I think this is the correct set https://github.com/jetstack/cert-manager/blob/dffbf391dbb0fc6c1cfea62e561a9c6f54362ab0/pkg/apis/certmanager/v1/types.go#L209